integration-cli/docker_cli_run_test.go

```diff
@@ -146,34 +146,6 @@ func (s *DockerSuite) TestRunWorkingDirectory(c *check.C) {
 	}
 }
 
-// pinging Google's DNS resolver should fail when we disable the networking
-func (s *DockerSuite) TestRunWithoutNetworking(c *check.C) {
```

Comment:
I'm not following.  Is `--net`/`-n` deprecated? I not then shouldn't this entire testcase be kept?